vendor/github.com/Shopify/sarama/.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 go:
 - 1.4.3
-- 1.5.2
```

Comment:
I was able to reproduce this in the old tree with:
```
godep restore
rm -rf Godeps
export  GO15VENDOREXPERIMENT=0
godep save ./...
```
I filed https://github.com/google/cadvisor/issues/1373 to prevent this in the future, but I think you can leave this as is.